interfaces/portal/src/app/pages/program-registration-debit-cards/components/link-card-on-site-dialog/link-card-dialog.component.html

```diff
@@ -0,0 +1,141 @@
+<p-dialog
+  [visible]="dialogVisible()"
+  modal
+  closable
+  dismissableMask
+  [style]="{ width: '632px' }"
```

Comment:
Would there be a safety-value for the `min-width`? Or a valid `max-width`?

That way, this dialog would became a lot more "safe for responsive use"...